bluepyemodel/emodel_pipeline/emodel_pipeline.py

```diff
@@ -131,6 +133,8 @@ def __init__(
                 "This variable should not be modified in new code."
             )
 
+        self.check_name(emodel)  # check if name complies with requirements
```

Comment:
The luigi workflow does not use EModel_pipeline, I believe. To be sure that the check is done everytime, maybe we should move the check at some other place, like e.e. https://github.com/BlueBrain/BluePyEModel/blob/main/bluepyemodel/emodel_pipeline/emodel_metadata.py#L57

Reply:
Good point, I will move it there

Reply:
I hope with the above comment location bpem will also check the nexus luigi pipeline or will there a need add such code there as well?

Reply:
Yes indeed, with Aurélien suggestion, the code should then be able to check it whether it is using the local or the luigi pipeline